msgraph/models.go

```diff
@@ -1685,3 +1685,25 @@ type EmployeeOrgData struct {
 	CostCenter *string `json:"costCenter,omitempty"`
 	Division   *string `json:"division,omitempty"`
 }
+
+type UserFlowType int
+
+const (
+	SIGNUP_USERFLOWTYPE UserFlowType = iota
+	SIGNIN_USERFLOWTYPE
+	SIGNUPORSIGNIN_USERFLOWTYPE
+	PASSWORDRESET_USERFLOWTYPE
+	PROFILEUPDATE_USERFLOWTYPE
+	RESOURCEOWNER_USERFLOWTYPE
+	UNKNOWNFUTUREVALUE_USERFLOWTYPE
+)
+
+type B2CUserFlow struct {
```

Comment:
B2cUserFlow has a bunch of fields https://learn.microsoft.com/en-us/graph/api/resources/b2cidentityuserflow?view=graph-rest-beta . I've added only the minimal ones. 

I can add more fields if required.

Reply:
We will want to add these in time, but I'm happy to merge with these fields for now.